Cargo.toml

```diff
@@ -64,8 +64,14 @@ version = "0.3.10"
 package = "errno"
 default-features = false
 
+# Rustix uses tempfile for its tests, which in turn uses rustix! We want to use
+# the same consistent version. We therefore need to use a version of tempfile
+# for which its rustix dependency is a semver match for our current version.
+[patch.crates-io]
+rustix = { path = "." }
```

Comment:
This will make it impossible to do a semver incompatible release of rustix as there is no tempfile version that has a semver compatible dependency on rustix yet.

Reply:
tempfile's use of rustix is not part of rustix's testsuite. We don't need to make tempfile's version of rustix match the version of rustix using tempfile for tests. They can differ and it's fine.

Reply:
So to be clear, I think it's best to omit this part of the patch.